### PR TITLE
[SGE] Phlegma Fixes

### PR DESCRIPTION
**Pull request by Tsusai**
PhlegmaReplacementFeature Bug Squishes:
Looked for the wrong GUI checkbox (fixes Nik-Potokar/XIVSlothCombo/#451)
Add "no-target change to Dyskrasia" logic

~~k-kz: Please test the GUI enable/disable is correct ( #451 ) and if Dyskrasia is enabled, it should show Dyskrasia when no target is selected.~~

### CODE REVIEW

**Comment by Tsusai** [comment]
Wrong branch.

File: XIVSlothCombo/Combos/SGE.cs [hunk 1]
```diff
@@ -195,7 +195,7 @@ protected override uint Invoke(uint actionID, uint lastComboMove, float comboTim
     //Can replace Zero Charges/Stacks of Phlegma with Toxikon (if you can use it) or Dyskrasia 
     internal class SagePhlegmaFeature : CustomCombo
     {
-        protected internal override CustomComboPreset Preset { get; } = CustomComboPreset.SagePhlegmaToxikonFeature;
+        protected internal override CustomComboPreset Preset { get; } = CustomComboPreset.SagePhlegmaFeature;
```
Comment (k-kz) [inline XIVSlothCombo/Combos/SGE.cs:198]: Have you accounted for this Feature name-change in `CustomComboPreset.cs`?
I don't see any changes to the file in this PR currently.

File: XIVSlothCombo/Combos/SGE.cs [hunk 2]
```diff
@@ -205,8 +205,15 @@ protected override uint Invoke(uint actionID, uint lastComboMove, float comboTim
                     (IsEnabled(CustomComboPreset.SagePhlegmaDyskrasiaFeature) && level >= SGE.Levels.Dyskrasia))
 
                 {
+                    //If not targetting anything, use Dyskrasia Option if selected
+                    if (IsEnabled(CustomComboPreset.SagePhlegmaDyskrasiaFeature))
+                    {
+                        if (!(HasTarget())) return OriginalHook(SGE.Dyskrasia);
+                    }
```
Comment (k-kz) [inline XIVSlothCombo/Combos/SGE.cs:212]: You can change this to
```if (IsEnabled(CustomComboPreset.SagePhlegmaDyskrasiaFeature) && !(HasTarget()))```
```return OriginalHook(SGE.Dyskrasia);```
and get rid of the { } entirely

**Review by k-kz — COMMENTED** [review]
Left comments on a few things that need to be addressed before merge.
Rest looks good!

**Pull request merged**